src/parser/sparqlParser/generated/SparqlAutomatic.g4

```diff
@@ -368,9 +368,13 @@ pathEltOrInverse
     ;
 
 pathMod
-    : '+' | '*' | '?'
+    : '+' | '*' | '?' | '{' stepsMin ( ',' ( stepsMax )? )? '}'
```

Comment:
The feature with the minimal and maximal steps was discussed in a previous draft of the property path spec:
[see here](https://www.w3.org/TR/sparql11-property-paths/)
We could use the exact syntax described there, which is almost what you have written here, but i think the case
`<p>{, 4}` which is short for `<p>{0, 4}` is not yet handled by your grammar expression.
(I think unfortunately you need a completely separate almost redundant case in the grammar,
but the code that consumes it could still be simple.